manifests/instance.pp

```diff
@@ -18,6 +18,8 @@
 #   enable service (boolean)
 # [*systemd_service_type*]
 #   value for systemd service type
+# [*systemd_workdir*]
```

Comment:
What if I instead added a parameter for systemd_params or systemd_extra_params that the templates could iterate over?